spec/fixtures/hiera/hathitrust.yaml

```diff
@@ -59,7 +59,7 @@ hathitrust::networks::staff:
   - "%{alias('networks::one')}"
   - "%{alias('networks::two')}"
 
-nebula::profile::ssl_keypair::chain_crt: 'intermediate_ca.crt'
+nebula::profile::ssl_keypair::chain_crt: 'bad_cert.crt'
```

Comment:
With the removal of the expectation, I think this is now extraneous. Not harmful, but unused.

Reply:
yep, purged this in a later commit.